Quality Control OCA

<!-- Provide a general summary of the issue in the Title above.
Before opening the issue, please check that there isn't another issue addressing the same problem already created -->

## Module
Quality Control OCA 

## Describe the bug
After installing quality Control OCA addon on ODOO 16 I'm unable to edit product categories due to error.
In general I'm not developer so if You need some more info about configuration please forgive me and ask.

## To Reproduce
ODOO: 16.0-20230807 
Quality Control OCA: 16.0.1.0.0

Error:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/odoo/tools/cache.py", line 85, in lookup
    r = d[key]
  File "<decorator-gen-6>", line 2, in __getitem__
  File "/usr/lib/python3/dist-packages/odoo/tools/func.py", line 87, in locked
    return func(inst, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/tools/lru.py", line 34, in __getitem__
    a = self.d[obj]
KeyError: ('res.users', <function Users._has_group at 0x7f9c10e60af0>, 2, 'group_quality_control_user')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 1584, in _serve_db
    return service_model.retrying(self._serve_ir_http, self.env)
  File "/usr/lib/python3/dist-packages/odoo/service/model.py", line 133, in retrying
    result = func()
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 1611, in _serve_ir_http
    response = self.dispatcher.dispatch(rule.endpoint, args)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 1815, in dispatch
    result = self.request.registry['ir.http']._dispatch(endpoint)
  File "/usr/lib/python3/dist-packages/odoo/addons/website/models/ir_http.py", line 235, in _dispatch
    response = super()._dispatch(endpoint)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_http.py", line 154, in _dispatch
    result = endpoint(**request.params)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 697, in route_wrapper
    result = endpoint(self, *args, **params_ok)
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/dataset.py", line 42, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/dataset.py", line 33, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 457, in call_kw
    result = _call_kw_model(method, model, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 430, in _call_kw_model
    result = method(recs, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_ui_view.py", line 2491, in get_views
    result['views'] = {
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_ui_view.py", line 2492, in <dictcomp>
    v_type: self.get_view(
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_ui_view.py", line 2679, in get_view
    node = self.env['ir.ui.view']._postprocess_access_rights(node)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_ui_view.py", line 1059, in _postprocess_access_rights
    if attrib_groups and not self.user_has_groups(attrib_groups):
  File "/usr/lib/python3/dist-packages/odoo/models.py", line 1470, in user_has_groups
    if user.has_group(group_ext_id):
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/res_users.py", line 969, in has_group
    return self._has_group(group_ext_id)
  File "<decorator-gen-111>", line 2, in _has_group
  File "/usr/lib/python3/dist-packages/odoo/tools/cache.py", line 90, in lookup
    value = d[key] = self.method(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/res_users.py", line 982, in _has_group
    assert group_ext_id and '.' in group_ext_id, "External ID '%s' must be fully qualified" % group_ext_id
AssertionError: External ID 'group_quality_control_user' must be fully qualified

The above server error caused the following client error:
RPC_ERROR: Odoo Server Error
    at makeErrorFromResponse (https://cadexo.pl/web/assets/5017-325edac/web.assets_backend.min.js:985:163)
    at XMLHttpRequest.<anonymous> (https://cadexo.pl/web/assets/5017-325edac/web.assets_backend.min.js:993:13)

--- a/quality_control_oca/views/product_category_view.xml
+++ b/quality_control_oca/views/product_category_view.xml
@@ -12,7 +12,7 @@
         <field name="inherit_id" ref="product.product_category_form_view" />
         <field name="arch" type="xml">
             <group name="first" position="after">
-                <t groups="group_quality_control_user">
+                <t groups="quality_control_oca.group_quality_control_user">
                     <group name="qc" string="Quality control">
                         <field name="qc_triggers" nolabel="1">
                             <tree editable="bottom">

